pyproject.toml

```diff
@@ -20,10 +20,10 @@ keywords = ["django", "services", "octue", "twined"]
 packages = [{ include = "django_twined" },]
 
 [tool.poetry.dependencies]
-python = ">=3.9,<4"
-django = ">=3.0,<5.0"
+python = ">=3.10,<4"
+django = ">=3.0,<5.1"
 octue = ">=0.52.0,<1"
-django-gcp = ">=0.11.5,<0.13"
+django-gcp = "^0.13.0"
```

Comment:
@Anusha0521 this is now quite restrictive. In semantic versioning, django-gcp moving from 0.11 to 0.12 or 0.13 represents a breaking change. A user of django-twined who wants to update will now also have to update django-gcp, migrating across any breaking changes. Forcing them to do that right now is a bit rough!

This PR doesn't make any change to the way in which django-gcp is used so it should still be backward compatible to `>=0.11.5`